src/effects/native/reverbeffect.cpp

```diff
@@ -27,36 +27,61 @@ EffectManifest ReverbEffect::getManifest() {
     time->setId("bandwidth");
     time->setName(QObject::tr("Bandwidth"));
     time->setShortName(QObject::tr("BW"));
-    time->setDescription(QObject::tr("Higher bandwidth values cause more "
-            "bright (high-frequency) tones to be included"));
+    time->setDescription(QObject::tr("Bandwidth of the low pass filter at the input. "
+            "Higher values result in less attenuation of high frequencies."));
     time->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
     time->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     time->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
-    time->setDefaultLinkType(EffectManifestParameter::LINK_LINKED);
-    time->setDefaultLinkInversion(EffectManifestParameter::LinkInversion::INVERTED);
-    time->setMinimum(0.0005);
-    time->setDefault(0.5);
-    time->setMaximum(1.0);
+    time->setMinimum(MIN_VALUE);
+    time->setDefault(MAX_VALUE);
+    time->setMaximum(MAX_VALUE);
+
+    EffectManifestParameter* decay = manifest.addParameter();
+    decay->setId("decay");
+    decay->setName(QObject::tr("Decay"));
+    decay->setDescription(QObject::tr("Lower decay values cause reverberations to die out more quickly."));
+    decay->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    decay->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    decay->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    decay->setMinimum(MIN_VALUE);
+    decay->setDefault(DEFAULT_DAMPING);
+    decay->setMaximum(MAX_VALUE);
 
     EffectManifestParameter* damping = manifest.addParameter();
     damping->setId("damping");
     damping->setName(QObject::tr("Damping"));
     damping->setDescription(QObject::tr("Higher damping values cause "
-            "reverberations to die out more quickly."));
+            "high frequencies to decay more quickly than low frequencies."));
     damping->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
     damping->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     damping->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
-    damping->setMinimum(0.005);
-    damping->setDefault(0.5);
-    damping->setMaximum(1.0);
+    damping->setMinimum(MIN_VALUE);
+    damping->setDefault(MIN_VALUE);
+    damping->setMaximum(MAX_VALUE);
+
+    EffectManifestParameter* blend = manifest.addParameter();
```

Comment:
The design of the chain itself is as an "insert" effect, not as a "send" effect, so dry/wet makes more sense.  I'm doing it the exact same way as the original LADSPA plugin which makes it easier to verify the correctness, and I think it's common to have wet/dry on a reverb effect intended as an insert. For example see pantheon:

https://www.kvraudio.com/product/pantheon-by-lexicon

Most traditional insert effects like eq don't require a wet/dry knob so I don't think it should be required.